src/manifolds/VectorBundle.jl

```diff
@@ -538,8 +369,162 @@ Notation:
 The projection is calculated by projecting $\xi_{v,M}$ to tangent space $T_{p_x}M$
 and then projecting the vector $\xi_{v,F}$ to the fiber $F$.
 """
+project_tangent(::VectorBundle, ::Any...)
 function project_tangent!(B::VectorBundle, w, x, v)
     project_tangent!(B.M, w.parts[1], x.parts[1], v.parts[1])
     project_tangent!(B.M, w.parts[2], x.parts[1], v.parts[2])
     return w
 end
+
+"""
+    project_vector(B::VectorBundleFibers, x, w)
+
+Project vector `w` from the vector space of type `B.VS` at point `x`
+and save the result to `v`.
+"""
+function project_vector(B::VectorBundleFibers, x, w)
```

Comment:
I think you should leave the erroring default version of `project_vector` (it's for vector spaces other than the tangent space)

Reply:
This one is similar to `exp` and `exp!` (see https://github.com/JuliaNLSolvers/ManifoldsBase.jl/blob/master/src/ManifoldsBase.jl#L337 ), where the non-mutating falls back to the mutating one and (ony) this throws the error.

Reply:
Sorry, I meant the mutating erroring variant:
```julia
"""
    project_vector!(B::VectorBundleFibers, v, x, w)
Project vector `w` from the vector space of type `B.VS` at point `x`
and save the result to `v`.
"""
function project_vector!(B::VectorBundleFibers, v, x, w)
    error("project_vector! not implemented for vector space family of type $(typeof(B)), output vector of type $(typeof(v)) and input vector at point $(typeof(x)) with type of w $(typeof(w)).")
end
```
It's still needed.

Reply:
Ah, I might have forgotten the more general one, since I might have mistaken the special one (tangent) to be the mutating one and accidentally deleted your general one (I'll omit the docs however).